interfaces/system_key_test.go

```diff
@@ -62,6 +64,12 @@ func (s *systemKeySuite) SetUpTest(c *C) {
 	c.Assert(err, IsNil)
 	s.buildID = id
 
+	s.seccompCompilerVersion = "123 2.3.3 abcdef123"
+	testutil.MockCommand(c, filepath.Join(dirs.DistroLibExecDir, "snap-seccomp"), fmt.Sprintf(`
```

Comment:
Thinking about this I was wondering if we should add `MustShellcheck(string) string` that panics if the string did not pass shellcheck. We could use it for testing like this.

Reply:
Sounds like an enhancement to MockCommand. Though a follow-up PR.